### DISCUSSION TITLE
Show dots horizontally for each industry with a count greater than 2

### DISCUSSION THREAD

**Discussion by winner106** [observablehq/plot#1605] [Q&A]
I  want to display dots horizontally for each industry, but only for those industries that have a count greater than 2. 

I have tried the following code: 

```js
Plot.dot(
  data,
    Plot.dodgeX(
      Plot.groupY(
        {x: 'count', filter: (d) => d.length > 2},
        {y: 'industry', r:"assets", fill:"sex", sort: {y: 'x', reverse: true}}
    )
  )
)
```
I would greatly appreciate any help. Thanks!

**Comment by Fil** [comment]
Can you share a dataset of **industries** that have a **sex** column? :D

If —ideally— you could share a notebook with code and data, it would help identify where you're hitting a difficulty.

**Comment by winner106** [comment]
A leaderboard of wealthy individuals, Sample data, Thanks
```json
 [ {
    "assets": 77.6,
    "year": 2023,
    "sex": "man",
    "name": "Elon Musk",
    "rank": 10,
    "company": "Tesla",
    "industry": "tech",
    "id": 152477,
  },
...
]
```

**Comment by Fil** [comment]
The issue might be that the dodgeX transform applies on one-dimensional data (it derives a channel X from a single channel Y), whereas groupY({x: 'count'}, {y: 'industry'}) creates both X and Y.

**Reply by winner106** [reply]
how can I fix it? I  want to Show dots horizontally for each industry that its count greater than 2

**Reply by Fil** [reply]
If you wanted to share a notebook with the dataset, it would be easier to understand where you block, and try and help.
